### DISCUSSION TITLE
Jamulus2.0 on Apple App Store

### DISCUSSION THREAD

**Discussion by rdica** [jamulussoftware/jamulus#2581] [Technical Support & Help] [answered]
Found this today https://apps.apple.com/us/app/jamulus2-0/id1609844773 on Apple App Store.
Anyone know anything about the developers, or know if this is an officially supported release?

**Comment by hoffie** [comment, marked as answer]
@gilgongo had tried reaching out to them a few days ago with no success so far.
I'm not too happy about non-official, far-reaching yet non-specialized publishing of Jamulus binaries outside of jamulus.io, especially without any mention that it is a custom build. Therefore, I would have liked if it was possible to work together to turn this into something official.

To clarify: The App Store is certainly far-reaching in that it has a large audience. It's non-official as we don't know the exact code base, the build process or the people behind it. It's also non-specialized in that it targets a whole platform, yet it is not done by those responsible for the platform (i.e. Apple itself). A more specific counter-example: I'm fine with Debian maintainers publishing binaries for Debian as that's both targeted (Debian-only) and transparent (open build process/patches. Same for other distributions of any kind.

**Comment by gilgongo** [comment]
Indeed - no response from them in two weeks (we've contacted their support email in English and Chinese) so we assume they don't intend to respond. 

We were intending to put Jamulus in the Appstore once we were ready to, and I don't suppose the existence of their version is necessarily a problem for that. But it's hard to say.

**Comment by chrisrimple** [comment]
I assume that there's a process for reporting problem apps to Apple.  I wonder if what's in the App Store violates any aspects of the GPL/other open source licenses in place for Jamulus?  If so, then it should be possible to log a take-down request with Apple.

**Reply by gilgongo** [reply]
I don't think the existence of their app is necessarily a problem for us though unless Apple say it is. As for the GPL stuff, we are intending to bypass that with [our waiver](https://github.com/jamulussoftware/jamulus/blob/master/APPLEAPPSTORE.LICENCE.WAIVER).

**Reply by ghost** [reply]
Jamulus was GPLv2 before any waiver, you cannot arbitrarily change the license with a waiver. The waiver may need to be legally inspected if it is too general or complicated or controversial or contravenes GPLv2 or whatever.

**Reply by gilgongo** [reply]
The licence has not changed. But if you want to investigate that, feel free to do so.

**Reply by ghost** [reply]
I read the waiver and the Apple link given. I did not find any reason for having a waiver here. What I did find is that Apple is talking about an EULA or End User License Agreement. The GPLv2 is not an End User License Agreement, it is a source-code/executable-code distributing agreement. If anyone, end user or otherwise, wishes to distribute executable code that was compiled with any Jamulus source code, the modified Jamulus source code must be provided... and must be compiled as separate linkable libraries (e.g. .dll MSFT or .so Unix) so that the libraries can be used by a proprietary executable. Otherwise, the executable will have to provide GPLv2 or later source code.
Note that I did not investigate whether Jamulus (GPLv2 or later) license allows you to use it as a library, if not, then ALL code WILL be (GPLv2 or later) and executibles SHALL comply or NOT be distributed.

**Comment by gene96817** [comment]
There is really two options.
First, does this unofficial release violate any license terms? If not just ignore it.

Second, when we are ready to list an official version, we just need to clearly state there is an official release. At that point, users will choose., I expect that the unofficial release will become moot. (unless they created some really useful new features.... effectively forking the Jamulus community) 

An official release in the Apple app store would be awesome.

**Comment by ghost** [comment]
If Jamulus2 is an Apple native Jamulus fork; this may be good because Jamulus2 will support Apple without the ever changing Apple bit rot affecting Jamulus.
Demand Jamulus2 source code is provided... by Apple(tm) if necessary. This will reduce the workload of maintaining Jamulus. Instead of starting an arms race.
